packages/elements-core/src/utils/http-spec/IServer.ts

```diff
@@ -53,7 +53,7 @@ export const getServerVariables = (server?: IServer | null): ServerVariable[] =>
 };
 
 export const getServerVariableDefaults = (server: IServer): Record<string, string> => {
-  const o = {};
+  const o: { [key: string]: string } = {};
```

Comment:
```suggestion
  const o: Record<string, string> = {};
```

Reply:
Done in dc54631ee990f5b015634c93e8449af47b27c366